### PR TITLE
Added support to publish outages as metrics

### PR DESCRIPTION
**Pull request by EldoreiJK**

### CODE REVIEW

File: include/fty-outage-server.h [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
-    it under the terms of the GNU General Public License as published by   
-    the Free Software Foundation; either version 2 of the License, or      
-    (at your option) any later version.                                    
-                                                                           
-    This program is distributed in the hope that it will be useful,        
-    but WITHOUT ANY WARRANTY; without even the implied warranty of         
-    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the          
-    GNU General Public License for more details.                           
-                                                                           
+    Copyright (C) 2014 - 2017 Eaton
```
Comment (jana-rapava) [inline include/fty-outage-server.h]: Update copyright please

File: src/data.h [hunk 2]
```diff
@@ -26,6 +26,7 @@
 
 // it is used as TTL, but in formula we are waiting for ttl*2 ->
 // so if we here would have 15 minutes-> the first alert will come in 30 minutes
+// TODO FIXME this is not respecting shm polling time, so if this gets lower than shm polling time false errors will be published
```
Comment (jana-rapava) [inline src/data.h]: Couldn't SHM polling time be published in some public header so we can read it here?

File: include/fty-outage-server.h (hunk 1 above)
Comment (EldoreiJK) [reply include/fty-outage-server.h]: ack

File: src/data.h (hunk 2 above)
Comment (EldoreiJK) [reply src/data.h]: That's already done, fty_get_polling_interval IIRC, or something like that, it's just not used and as discussed with Clement, should be just tagged as this agent is next in line for refactoring

Comment (jana-rapava) [reply src/data.h]: Ack, thanks for info

**Pull request merged**